lib/Graph/Graph.cpp

```diff
@@ -407,6 +407,10 @@ Constant *Module::createConstant(llvm::StringRef name, const Tensor &tensor) {
   return V;
 }
 
+Constant *Module::createConstant(llvm::StringRef name, Tensor &&tensor) {
+  return addConstant(new Constant(name, std::move(tensor)));
```

Comment:
consider forward vs move:
  return addConstant(new Constant(name, std::forward<Tensor>(tensor)));

Reply:
@putivsky Thank you for pointing this out. I am not sure that I understand the difference between the two. I was not able to find a good answer on google. Can you explain why you prefer std::forward over std::move?

Reply:
I don't think `std::forward` is applicable here because `Tensor&& tensor` is not a universal reference. It's not possible for an lvalue reference to bind to it. For example: https://godbolt.org/z/yQwHhR

Reply:
@SplitInfinity nice explanation

Reply:
In case people are curious, if this function was templated like so:
```
template <typename T>
Constant *Module::createConstant(llvm::StringRef name, T &&tensor) { ... }
```
then `std::forward` would be applicable here. I think you can still use it, but there's no point it will do the same thing that `std::move` does every time.